lib/msf/ui/console/command_dispatcher/auxiliary.rb

```diff
@@ -48,9 +55,25 @@ def method_missing(meth, *args)
 
       return mod.send(meth.to_s, *args)
     end
+
+    action = meth.to_s.delete_prefix('cmd_')
+    mod_actions = mod.actions.map(&:name)
+    if mod && mod.kind_of?(Msf::Module::HasActions) && mod_actions.include?(action)
```

Comment:
I'm a little concerned about the safety here of these two lines. On L61 you check if `mod` is set, but that's after accessing `#actions` on it. Additionally, since we're using lowercase command names this will need to be normalized for the comparison.

```
    if mod && mod.kind_of?(Msf::Module::HasActions) && mod.actions.map(&:name).map(&:downcase).include?(action)
```

This would make sure that we're not accessing things on `mod` if it's nil as well as ensure that the action name is checked in a case insensitive manner (by converting the values to lowercase).